charts/unleash-proxy/templates/deployment.yaml

```diff
@@ -93,6 +97,46 @@ spec:
               port: http
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
+        {{- if .Values.edge.enable }}
+        - name: unleash-edge
+          args:
+            - "offline"
+          env:
+            - name: TOKENS
+              value: {{ .Values.proxy.apiToken }}
+            - name: BOOTSTRAP_FILE
+              value: /data/config/features.json
+          image: "unleashorg/unleash-edge:v0.5.1"
+          imagePullPolicy: IfNotPresent
+          ports:
+            - name: http
+              containerPort: 3063
+              protocol: TCP
+          livenessProbe:
+            httpGet:
+              path: /internal-backstage/health
+              port: http
+          readinessProbe:
+            httpGet:
+              port: http
+              path: /internal-backstage/health
```

Comment:
Unrelated, but something to think about: I guess for offline mode ready and alive means the same, but if features are async (first request ends up in "come back in a few milliseconds"), maybe it makes sense to have a different endpoint to check readiness